PyInstaller/depend/bindepend.py

```diff
@@ -942,7 +942,9 @@ def _find_lib_in_libdirs(*libdirs):
         # and exec_prefix. That's why we can use just sys.prefix.
         # In virtualenv PyInstaller is not able to find Python library.
         # We need special care for this case.
-        python_libname = _find_lib_in_libdirs(compat.base_prefix)
+        python_libname = _find_lib_in_libdirs((
+            compat.base_prefix,
+            os.path.join(compat.base_prefix, 'lib')))
```

Comment:
I think there's one set of braces too many here, so you're passing to the function a single tuple with two paths instead of two paths as separate arguments (which are then collected into `libdirs` tuple due to `def _find_lib_in_libdirs(*libdirs)`).

Reply:
thank you, will correct it now, this works by chance as libdirs becomes the tuple so the for .. in still iterates.

Reply:
corrected